projects/tcpdump/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd libpcap
+mkdir build
+cd build
+cmake ..
+make
+make install
+
+cd ../../tcpdump
+# build project
+mkdir build
+cd build
+cmake ..
+make
+
+# build fuzz targets
```

Comment:
I'd rather see this be done as part of the tcpdump build process, so that the project can add fuzz targets without having to push a change to build.sh.

Reply:
If that helps, the fuzz targets are built with tcpdump as well, but with a different stand alone driver (onefile.c)

> so that the project can add fuzz targets without having to push a change to build.sh
This is indeed a useful feature when the need arises

Reply:
It's true that they are built as part of the tcpdump build process, but that is not useful to clusterfuzz.  They have to be built during this process using the fuzzer library to be able to be fuzzed in the cluster.

I have a Makefile.in that can build either tcpdump-way or clusterfuzz-way, but I don't know how to do it in cmake.  We can simply add the "build the fuzzers the clusterfuzz way" build.sh in /src/tcpdump/fuzz/build.sh and invoke that from here.